### DISCUSSION TITLE
Developing in the cloud?

### DISCUSSION THREAD

**Discussion by thatmichaelpark** [probonopd/MiniDexed#290] [Q&A]
The wiki [Development page](https://github.com/probonopd/MiniDexed/wiki/Development) mentions developing in the cloud, but when I click the green Code button [here](https://github.com/probonopd/MiniDexed), I don't get a Codespaces option.

Is this some sort of gatekeeping to keep incompetents like me out of trouble?

**Comment by probonopd** [comment]
Hello @thatmichaelpark. Possibly I had opted into this some time ago, don't quite remember, but here is more information about it: https://github.com/features/codespaces
